godtools/App/Services/Renderer/Views/MobileContent/Views/ContentFlowItem/MobileContentFlowItemViewModelType.swift

```diff
@@ -11,5 +11,6 @@ import GodToolsToolParser
 
 protocol MobileContentFlowItemViewModelType {
     
+    var visibilityState: ObservableValue<MobileContentViewVisibilityState> { get }
```

Comment:
hmm, would it be worthwhile to update parts of the shared parser to produce `ObservableValue` wrappers around the various coroutines flows to make it easier to digest than a watcher on your end?

Reply:
I'm assuming `ObservableValue` is a core API and not from a 3rd party library

Reply:
Hey @frett let's hold off on that.  ObservableValue is a simple Class I created for observing a stored value and when it changes.  With our moving to iOS 13 the plan is to eventually phase out ObservableValue in place of Combine's CurrentValueSubject https://developer.apple.com/documentation/combine/currentvaluesubject